tests/integration/cocoon/test_seed_sample_data.py

```diff
@@ -0,0 +1,102 @@
+import unittest
+from pathlib import Path
+import pandas as pd
+import lusid
+from lusidtools.cocoon.seed_sample_data import seed_sample_data
+from lusidtools.cocoon.utilities import create_scope_id
+import logging
+import os
+import datetime
+import pytz
+import random
+
+logger = logging.getLogger()
+
+
+class CocoonTestPortfolioGroup(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+
+        cls.scope = create_scope_id()
+        secrets_file = Path(__file__).parent.parent.parent.joinpath("secrets.json")
+        cls.api_factory = lusid.utilities.ApiClientFactory(
+            api_secrets_filename=secrets_file
+        )
+        cls.sample_data = pd.read_csv(
+            os.path.join("data", "seed_sample_data", "sample_data.csv")
+        )
+
+        seed_sample_data(
+            cls.api_factory,
+            ["portfolios", "instruments", "transactions"],
+            cls.scope,
+            os.path.join("data", "seed_sample_data", "sample_data.csv"),
+        )
+
+    def test_transactions_from_response(self):
+
+        transactions_from_response = self.api_factory.build(
+            lusid.api.TransactionPortfoliosApi
+        ).get_transactions(
+            scope=self.scope,
+            code=self.sample_data["fund_code"].to_list()[0],
+            from_transaction_date=datetime.datetime.now(tz=pytz.UTC)
+            - datetime.timedelta(weeks=52),
+        )
+
+        self.assertEqual(
+            set([txn.transaction_id for txn in transactions_from_response.values]),
+            set(self.sample_data["txn_id"].to_list()),
+        )
+
+    def test_portfolio_from_response(self):
+
+        portfolio_from_response = self.api_factory.build(
+            lusid.api.PortfoliosApi
+        ).get_portfolio(
+            scope=self.scope, code=self.sample_data["fund_code"].to_list()[0]
+        )
+
+        self.assertEqual(
+            portfolio_from_response.id.code, self.sample_data["fund_code"].to_list()[0]
+        )
+
+    def test_bad_file_type(self):
+
+        with self.assertRaises(ValueError) as error:
+
+            seed_sample_data(
+                self.api_factory,
+                ["bad_file_type"],
+                self.scope,
+                os.path.join("data", "seed_sample_data", "sample_data.csv"),
+            )
+
+        self.assertEqual(
+            error.exception.args[0],
+            "The provided file_type of bad_file_type has no associated mapping",
+        )
+
```

Comment:
we should add a test for missing columns for valid file types

Reply:
Cocoon already naively handles that scenario with a nice error.
Example:
```
ValueError: The values {'txn_type'} exist in the mapping_required
                                   but do not exist in the DataFrame Columns.
```
What do you think?

Reply:
yep that works